### PR TITLE
Namespace images with sha collisions during sync 

### PR DESCRIPTION
**Pull request by rnathuji**
**NOTE**: While the code related changes are ready for review, the content changes aren't reflected yet. I can add a commit to update the actual content either to this PR or a separate one, but I'd like to do that after [this PR](https://github.com/openstax/ce-git-storage-spike/pull/5) has been merged since that includes a switch to use production versus staging when fetching with `neb`.

### CODE REVIEW

File: consolidate_media.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    ]
+
+    print(f"{len(media_files)} media files found")
+
+    # Pass 1: Find any files that have sha collisions so we know how to handle
+    # them when we move to the media directory
+
+    shas_by_filename = {}  # Map of filename: sha
+    shas_by_filepath = {}  # Map of filepath: sha
+    sha_collisions = set()  # Set of file names with sha collisions
```
[the remark this answers is not in the document]
Comment (tomjw64) [reply consolidate_media.py:49]: For the same reason, I feel the log message on L65 is also a bit misleading.

[the remark this answers is not in the document]
Comment (rnathuji) [reply consolidate_media.py:49]: Oh, I see --- maybe this should be `sha_conflicts`. Is that more clear?

[the remark this answers is not in the document]
Comment (tomjw64) [reply consolidate_media.py:49]: I think either `filename_collisions` or `sha_conflicts` makes more sense.

[the remark this answers is not in the document]
Comment (rnathuji) [reply consolidate_media.py]: [Done](https://github.com/openstax/ce-git-storage-spike/pull/6/commits/695741755fca58cfb40caddf45d48122433036b6)

**Review by tomjw64 — APPROVED** [review]

**Pull request merged**